### PR TITLE
Embroider and glimmer logos

### PR DESCRIPTION
**Pull request by jeffdaley**

### CODE REVIEW

**Review by wifelette — APPROVED** [review]
Approving _conceptually_, but someone should approve the actual _code_ :p

**Review by locks — APPROVED** [review]

**Review by mansona — APPROVED** [review]
looks great! can you rebase on master first before we merge so that https://github.com/ember-learn/ember-website/pull/774 commits aren't also in this PR 👍

**Pull request merged**
